src/components/AddBook/index.js

```diff
@@ -11,6 +11,8 @@ function AddBook() {
   const [imgUrl, setImgUrl] = useState("");
   const [blurb, setBlurb] = useState("");
 
+  const [bookGenres, setBookGenres] = useState([]);
+
   const [redirect, setRedirect] = useState(false);
 
   const [titleErrors, setTitleErrors] = useState("");
```

Comment:
(wont let me comment on the correct lines, but please see my comment on AddReviewForm about your cool change handler, please apply that approach here as well